arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -67,13 +67,14 @@ class ffmpeg_video_input::priv
     video_path(""),
     filter_desc("yadif=deint=1"),
     metadata(0),
-    frame_advanced(0),
+    frame_advanced(false),
     end_of_video(true),
     number_of_frames(0),
     collected_all_metadata(false),
     estimated_num_frames(false)
   {
-    f_packet.data = nullptr;
+    // Allocate data
+    f_format_context = avformat_alloc_context();
```

Comment:
Could we just initialize this in the initializer list above?  Previously it was `f_format_context(nullptr),`, but now it could be `f_format_context(avformat_alloc_context()),`.

Reply:
Done